src/middlewared/middlewared/plugins/webshare/config.py

```diff
@@ -51,10 +51,8 @@ async def do_update(self, data):
     @private
     def setup_directories(self):
         os.makedirs(WEBSHARE_BULK_DOWNLOAD_PATH, mode=0o700, exist_ok=True)
-        os.chown(WEBSHARE_BULK_DOWNLOAD_PATH, WEBSHARE_UID, WEBSHARE_GID)
```

Comment:
Is webshare running with privileged capabilities? How is it bypassing the permissions here?